src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.Process.cs

```diff
@@ -58,8 +58,16 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             if (!File.Exists(HostRunner))
                 throw new IOException($"{HostRunner} test app isn't present in the test runtime directory.");
 
-            psi.FileName = HostRunner;
-            psi.Arguments = testConsoleAppArgs;
+            if (options.RunAsSudo)
```

Comment:
>if (options.RunAsSudo) [](start = 12, length = 22)

I suggest testing if it is not Windows too. or make the setter of RunAsSudo to throw if set for Windows

Reply:
PlatformNotSupportedException...

Reply:
I'll change it to throw an exception if you try to set `RunAsSudo` on Windows.